### PR TITLE
Staging 20180704

### PR DESCRIPTION
**Pull request by gctucker**
Summary:

* Fix bug with multiple email recipients in bisection reports
* Use new buildroot `kci-2018.05`
* Add igt test plan for drm and run on veyron-jaq and odroid-xu3
* Add cros-ec test plan and run on veyron-jaq and peach-pi
* Use `/dev/kmsg` with sleep and v4l2 test plans

### CODE REVIEW

**Comment by gctucker** [comment]
Tested on staging:
https://staging.kernelci.org/build/gtucker/branch/kernelci-test/kernel/kernelci-test-070-2-gf61cdef12369/
https://staging.kernelci.org/boot/all/job/gtucker/branch/kernelci-test/kernel/kernelci-test-070-2-gf61cdef12369/
https://staging.kernelci.org/test/board/rk3288-veyron-jaq/job/gtucker/kernel/kernelci-test-070-2-gf61cdef12369/

File: templates/sleep/sleep.jinja2 [hunk 1]
```diff
@@ -13,9 +13,9 @@
           - functional
         run:
           steps:
-            - dmesg -n 6  # Silence KERN_INFO as it confuses LAVA on wakeup
             - for i in $(seq 1 10); do lava-test-case rtcwake-mem-$i --shell rtcwake -d rtc0 -m mem -s 1; done
             - for i in $(seq 1 10); do lava-test-case rtcwake-freeze-$i --shell rtcwake -d rtc0 -m freeze -s 1; done
+      lava-signal: kmsg
```
Comment (mattface) [inline templates/sleep/sleep.jinja2:18]: This isn't merged yet, and for various reasons doesn't actually work. Maybe better to leave the dmesg call?

Comment (gctucker) [reply templates/sleep/sleep.jinja2:18]: Yes it's merged: https://review.linaro.org/#/c/25377/
and applied in `lab-collabora`.  As we control in which labs which test plans are run, this should be fine.

Comment (mattface) [reply templates/sleep/sleep.jinja2:18]: Hah, I leave LAVA for a week... :)
Ok cool, I'll merge this

**Pull request merged**